src/app/options/options.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-card>
+  <mat-card-title>Options</mat-card-title>
```

Comment:
Will this component serve any other purpose other than to display the word "Options?".  If not, its unnecessary to create a whole component dedicated to it.  Can you add the "Options" title to the App Component itself?

Reply:
Disregard, I see you do make use of it in subsequent PR